src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -39,34 +41,31 @@ limitations under the License.
     <field name="Saturday" mask="0x40"/>
     <field name="Away" mask="0x80"/>
   </bitmap>
-
-  <bitmap name="ModeForSequence" type="bitmap8">
+  <bitmap name="ScheduleModeBitmap" type="bitmap8">
     <cluster code="0x0201"/>
     <field name="HeatSetpointPresent" mask="0x01"/>
     <field name="CoolSetpointPresent" mask="0x02"/>
   </bitmap>
-
-  <enum name="ThermostatSystemMode" type="enum8">
-    <cluster code="0x0201"/>
-    <item name="Off" value="0x0"/>
-    <item name="Auto" value="0x1"/>
-    <item name="Cool" value="0x3"/>
-    <item name="Heat" value="0x4"/>
-    <item name="EmergencyHeat" value="0x5"/>
-    <item name="Precooling" value="0x6"/>
-    <item name="FanOnly" value="0x7"/>
-    <item name="Dry" value="0x8"/>
-    <item name="Sleep" value="0x9"/>
+  <enum name="SystemModeEnum" type="enum8">
+    <cluster code="0x0201"/>
+    <item name="Off" value="0x00"/>
+    <item name="Auto" value="0x01"/>
+    <item name="Cool" value="0x03"/>
+    <item name="Heat" value="0x04"/>
+    <item name="EmergencyHeat" value="0x05"/>
+    <item name="Precooling" value="0x06"/>
+    <item name="Fan only" value="0x07"/>
```

Comment:
I'll ask hasty since it's his tool that generated this.  The tool is intended to follow our current best practrices so that eventually we can just regenerate all the xml once the spec is approved and it'll be consistent (and a lot less work in the long term for all of us)